presto-spi/src/test/java/io/prestosql/spi/predicate/TestEquatableValueSet.java

```diff
@@ -111,7 +113,9 @@ public void testMultipleValues()
         assertFalse(equatables.isAll());
         assertFalse(equatables.isSingleValue());
         assertTrue(equatables.isWhiteList());
-        assertTrue(Iterables.elementsEqual(equatables.getValues(), ImmutableList.of(1L, 2L, 3L)));
+        Collection<Object> values = equatables.getValues();
+        assertEquals(values.size(), 3);
+        assertTrue(values.containsAll(ImmutableSet.of(1L, 2L, 3L)));
```

Comment:
why did the semantic change (from `elementsEqual` to `containsAll`)?

Reply:
When I wrote this code there was no ordering guarantee in `EquatableValueSet`.  I see that it is changed now, so I will revert this part.